presto-cache/src/main/java/com/facebook/presto/cache/filemerge/FileMergeCacheManager.java

```diff
@@ -140,6 +150,15 @@ public FileMergeCacheManager(
                 }
             }));
         }
+
+        this.cacheSizeCalculateExecutor.scheduleAtFixedRate(
```

Comment:
Nice, we changed it from schedule!

In general, do we know how many files exist in the cache directory? It would be worthwhile to know the impact of building this in every few seconds. I would have preferred to do this asynchronously whenever we write any files to cache.

Reply:
Actually we don't and also it should vary from different use case. I can try to see if we can add any indicator for us to tell the impact